resources/codicon.css

```diff
@@ -0,0 +1,474 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Can you remove the `See License.txt in the project root for license information.`
and add the full text of the MIT License in this comment?

Reply:
Also, can you add the link you sourced that file from?